less/index.less

```diff
@@ -4,56 +4,378 @@
+blockquote,
+q {
+  quotes: none;
 }
-blockquote:before, blockquote:after,
-q:before, q:after {
-	content: '';
-	content: none;
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+  content: "";
+  content: none;
 }
 table {
-	border-collapse: collapse;
-	border-spacing: 0;
+  border-collapse: collapse;
+  border-spacing: 0;
 }
 
 /* Set every element's box-sizing to border-box */
 * {
+  box-sizing: border-box;
+}
+
+html,
+body {
+  height: 100%;
+  font-family: "Archivo", sans-serif;
+  font-size: 62.5%;
+}
+
+// My Code Starts Here
+//Variables
+@primaryFontWeight: 400;
+@secondaryFontWeight: 600;
+@radius: 10px;
+@flexOnThem: flex;
+
+//Mixins
+
+.linkStyle {
+  border-bottom: 1px black dashed;
+  padding-top: 3px;
+  :hover {
+    color: #3b5998;
+  }
+}
+
+.box {
+  margin-top: 20px;
+  padding: 100px;
+  margin-bottom: 50px;
+  font-size: 3rem;
+  width: 100%;
+  border: none;
+  background: linear-gradient(to left, #ffffff, #ffd3337c);
+  box-shadow: 0px 0px 8px -2px rgba(0, 0, 0, 0.75);
+  border-radius: @radius;
+}
+
+.button {
+  background: lightblue;
+  font-size: 1.6rem;
+  color: #fff;
+  border-radius: 5px;
+  padding: 6px 4px;
+  text-decoration: none;
+}
+
+.container {
+  width: 800px;
+  height: 100vh;
+  margin: 0 auto;
+  @media (max-width: 800px) {
+    width: 100%;
+  }
+}
+
+header {
+  width: 100%;
+  nav {
+    padding-top: 20px;
+    display: flex;
+    justify-content: flex-end;
+
+    a {
+      text-decoration: none;
+      color: black;
+      margin-right: 20px;
+      font-size: 1.5rem;
+    }
+  }
+}
+
+.section-one {
+  background: #e7e3e39d;
+  width: 100%;
+  height: 400px;
+  margin-top: 30px;
+  border-radius: @radius;
+  display: @flexOnThem;
+  justify-content: center;
+  align-items: center;
+  .content {
+    h1 {
+      font-size: 5rem;
+    }
+    h3 {
+      font-size: 2.5rem;
+      text-align: center;
+      padding-top: 3px;
+    }
+  }
+
+  @media (max-width: 800px) {
+    width: 100%;
+    box-sizing: border-box;
+  }
+}
+
+.section-two {
+  display: flex;
+  width: 100%;
+  //   height: 200px;
+  justify-content: space-between;
+  align-items: flex-start;
+  .content {
+    width: 100%;
+    height: 300px;
+    @media (max-width: 800px) {
+      height: 200px;
+    }
+    h3 {
+      font-size: 2.4rem;
+      margin-top: 20px;
+      font-weight: @secondaryFontWeight;
+      @media (max-width: 800px) {
+        text-align: center;
+      }
+    }
+    p {
+      font-size: 1.5rem;
+      padding-top: 20px;
+      line-height: 3rem;
+      @media (max-width: 800px) {
+        text-align: center;
+      }
+    }
+
+    ul {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      margin-left: 20px;
+      @media (max-width: 800px) {
+        width: 50%;
+        flex-direction: row;
+        align-items: flex-end;
+      }
+      @media (max-width: 500px) {
+        width: 50%;
+        box-sizing: border-box;
+        align-items: center;
+      }
```

Comment:
You went above and beyond choosing to create mobile and tablet views! Nice work!